lib/util/git/index.ts

```diff
@@ -1028,14 +1030,14 @@ export async function pushCommit({
     }
 
     const pushRes = await gitRetry(() =>
-      git.push('origin', `${branchName}:${branchName}`, pushOptions)
+      git.push('origin', `${sourceRef}:${targetRef}`, pushOptions)
```

Comment:
```suggestion
      git.push('origin', `${sourceRef}:${targetRef ?? sourceRef}`, pushOptions)
```

So we can make `targetRef` optional (less code changes)